new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -533,46 +643,20 @@ public void testDomainInPvUsingWdt() {
     assertTrue(loginSuccessful, "Console login validation failed");
 
     logger.info("Getting node port for T3 channel");
-    int t3channelNodePort = assertDoesNotThrow(()
-        -> getServiceNodePort(wdtDomainNamespace, adminServerPodName + "-external", "t3channel"),
+    int t3channelNodePort = assertDoesNotThrow(() -> getServiceNodePort(
+        wlstDomainNamespace, adminServerPodName + "-external", "t3channel"),
         "Getting admin server t3channel node port failed");
     assertNotEquals(-1, t3ChannelPort, "admin server t3channelport is not valid");
 
-    //create ingress controller
-    Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
-    clusterNameMsPortMap.put(clusterName, managedServerPort);
-    logger.info("Creating ingress for domain {0} in namespace {1}", domainUid, wdtDomainNamespace);
-    createIngressForDomainAndVerify(domainUid, wdtDomainNamespace, clusterNameMsPortMap);
-
-    //deploy application
+    //deploy application using t3 channel port
     Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
-    logger.info("Deploying webapp to domain {0}", archivePath);
+    logger.info("Deploying webapp {0} to domain using t3 channelport", archivePath);
     DeployUtil.deployUsingWlst(K8S_NODEPORT_HOST, Integer.toString(t3channelNodePort),
         ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT, clusterName + "," + adminServerName, archivePath,
-        wdtDomainNamespace);
-
-    //access application from admin server
-    String url = "http://" + K8S_NODEPORT_HOST + ":" + serviceNodePort + "/testwebapp/index.jsp";
-    assertEquals(200,
-        assertDoesNotThrow(() -> OracleHttpClient.get(url, true),
-            "Accessing sample application on admin server failed")
-            .statusCode(), "Status code not equals to 200");
-
-    //access application in managed servers through NGINX load balancer
-    logger.info("Accessing the sample app through NGINX load balancer");
-    String curlRequest = String.format("curl --silent --show-error --noproxy '*' "
-        + "-H 'host: %s' http://%s:%s/testwebapp/index.jsp",
-        domainUid + "." + clusterName + ".test", K8S_NODEPORT_HOST, nodeportshttp);
-    List<String> managedServers = new ArrayList<>();
-    for (int i = 1; i <= replicaCount; i++) {
-      managedServers.add(domainUid + "-" + managedServerNameBase + i);
-    }
-    assertThat(callWebAppAndCheckForServerNameInResponse(curlRequest, managedServers, 20))
-        .as("Verify NGINX can access the test web app from all managed servers in the domain")
-        .withFailMessage("NGINX can not access the test web app from one or more of the managed servers")
-        .isTrue();
+        wlstDomainNamespace);
```

Comment:
The application is not accessed after deployment for this domain.

Reply:
for that load balancer is needed, then we have move everything into one test.